plugins/glustershd/rest.go

```diff
@@ -1,197 +1,129 @@
 package glustershd
 
 import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
 	"net/http"
+	"os/exec"
+	"path"
+	"strings"
 
 	"github.com/gluster/glusterd2/glusterd2/gdctx"
 	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
 	"github.com/gluster/glusterd2/glusterd2/transaction"
 	"github.com/gluster/glusterd2/glusterd2/volume"
-	"github.com/gluster/glusterd2/pkg/errors"
+	gderrors "github.com/gluster/glusterd2/pkg/errors"
+	glustershdapi "github.com/gluster/glusterd2/plugins/glustershd/api"
 
 	"github.com/gorilla/mux"
-	"github.com/pborman/uuid"
-	log "github.com/sirupsen/logrus"
+	config "github.com/spf13/viper"
 )
 
-func isVolReplicate(vType volume.VolType) bool {
-	if vType == volume.Replicate || vType == volume.Disperse || vType == volume.DistReplicate || vType == volume.DistDisperse {
-		return true
-	}
-
-	return false
-}
-
-func glustershEnableHandler(w http.ResponseWriter, r *http.Request) {
-	// Implement the help logic and send response back as below
-	p := mux.Vars(r)
-	volname := p["name"]
-
-	ctx := r.Context()
-	logger := gdctx.GetReqLogger(ctx)
+var (
+	healInfoOutput string
```

Comment:
It's not good to use global variables in REST handlers. Why is a global variable needed ?